### PR TITLE
Feature/jm 7139 new sign in

### PR DESCRIPTION
**Pull request by CadeFaulkner**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by CadeFaulkner — COMMENTED** [review]
Awaiting backend and frontend changes / discussion

File: src/main/java/cucumber/pageObjects/Bureau.java [hunk 1]
```diff
@@ -142,5 +142,9 @@ public void pressBackLink() throws Exception {
 			}
 		}
 	}
+	public void clickChangeLink(){
+		WebElement changeLink = driver.findElement(By.xpath("//*[contains(text(), 'Change')][not(@href='/auth/courts-list')]"));
```
Comment (lukesmithcgi) [inline src/main/java/cucumber/pageObjects/Bureau.java:146]: Why does this class not use @FindBy annotations like the other page objects?

File: src/test/java/cucumber/steps/StepDef.java [hunk 2]
```diff
@@ -100,10 +108,96 @@ public void loadPage_for_Welsh_Expenses(String publicBureau, String env) throws
...
+			case "CMANAGER2":
+			case "SHREWSBURY":
+			case "NEWUSER":
+				LGN.loginADTestRoute(username + "@email.gov.uk");
+				break;
+			case "MODTESTCOURT":
+				defaultCourt = "415";
+				LGN.loginADTestRoute(username + "@email.gov.uk");
+				break;
+			case "MODCOURT":
```
Comment (lukesmithcgi) [inline src/test/java/cucumber/steps/StepDef.java:144]: Don't take my word for this but I think you can probably do:

```
case "MODTESTCOURT": case "MODCOURT":
	defaultCourt = "415";
	LGN.loginADTestRoute(username + "@email.gov.uk");
	break;
```

**Review by lukesmithcgi — CHANGES_REQUESTED** [review]

File: src/main/java/cucumber/pageObjects/Bureau.java (hunk 1 above)
Comment (CadeFaulkner) [reply src/main/java/cucumber/pageObjects/Bureau.java:146]: I'll change this, I think initially I was intending on doing something different but changed my mind

File: src/test/java/cucumber/steps/StepDef.java (hunk 2 above)
Comment (CadeFaulkner) [reply src/test/java/cucumber/steps/StepDef.java:144]: wasn't sure if different users were needed to have different default courts, so left this as a bit of an example for that, although probably should have added some comments to explain that

File: src/test/resources/db/001_users.sql [hunk 3]
```diff
@@ -7,25 +7,27 @@
...
-       ('400', 'BUREAU', 'CPASS', 'CPASS@email.gov.uk', 'CPASS', true, 1, 0, NULL, NULL, 'MODTESTBUREAU', 'MODTESTBUREAU'),
-       ('400', 'BUREAU', 'NEWUSER', 'NEWUSER@email.gov.uk', 'NEWUSER', true, 1, 0, NULL, NULL, 'MODTESTBUREAU', 'MODTESTBUREAU'),
-       ('400', 'BUREAU', 'ARAMIS1', 'ARAMIS1@email.gov.uk', 'ARAMIS1', true, 1, 0, NULL, NULL, 'MODTESTBUREAU', 'MODTESTBUREAU'),
-       ('415', 'COURT', 'CMANAGER', 'CMANAGER@email.gov.uk', 'CMANAGER', true, 1, 1000, NULL, NULL, 'MODTESTBUREAU', 'MODTESTBUREAU'),
-       ('415', 'COURT', 'CMANAGER2', 'CMANAGER2@email.gov.uk', 'CMANAGER2', true, 1, 1000, NULL, NULL, 'MODTESTBUREAU', 'MODTESTBUREAU'),
-       ('452', 'COURT', 'SHREWSBURY', 'SHREWSBURY@email.gov.uk', 'SHREWSBURY', true, 1, 0, NULL, NULL, 'MODTESTBUREAU', 'MODTESTBUREAU'),
-       ('400', 'ADMINISTRATOR', 'SYSTEMADMIN', 'SYSTEMADMIN@email.gov.uk', 'SYSTEMADMIN', true, 1, 0, NULL, NULL, 'MODTESTBUREAU', 'MODTESTBUREAU'),
-       ('415', 'COURT', 'MODCOURT', 'MODCOURT@email.gov.uk', 'MODCOURT', true, 1, 0, NULL, NULL, 'MODTESTBUREAU', 'MODTESTBUREAU');
+INSERT INTO juror_mod.users (user_type, username, email, name, active)
```
Comment (lukesmithcgi) [inline src/test/resources/db/001_users.sql:10]: created_by is now mandatory

**Review by lukesmithcgi — CHANGES_REQUESTED** [review]

**Review by lukesmithcgi — APPROVED** [review]

**Pull request merged**